tests/integration/integration_steps/cdf_steps.py

```diff
@@ -254,7 +254,7 @@ def delete_state_store_in_cdf(
             if row is not None:
                 cognite_client.raw.rows.delete(database, table, statename)
 
-    all_rows = cognite_client.raw.rows.list(database, table, limit=1)
+    all_rows = cognite_client.raw.rows.list(database, table)
```

Comment:
Could this call fail? Are we good with that?

Reply:
I added a story to remove the custom code for removing the state store data.  We should conform to deleting state using the state store object.